src/chrome/content/rules/PostgreSQL.xml

```diff
@@ -1,49 +1,42 @@
 <!--
-	For problematic rules, see PostgreSQL-self-signed.xml.
+	Other PostgreSQL rulesets:
+		- Postgresql.eu.xml
+		- Postgresql.fr.xml
+		- Postgresql.us.xml
 
+	As of 2018-06-23, known unprotected .org subdomains are:
 
-	As of 2012-02-09, known unprotected .org subdomains are:
-
-		- archives
-		- babel
-		- buildfarm
-		- doxygen
-		- ftp
-		- git
-		- jdbc
-		- media
 		- *.projects
-		- webmail
-		- www
-		- yum
+		- nagios.postgresql.org (mismatch)
 
 
 	Other nonfunctional domains:
 
 		- (www.)pgfoundry.org
 		- (www.)pgsql.ru
 		- images.pgsql.ru
-		- docs.postgresql.fr *
-		- www.postgresql.fr *
-
-	* Shows login; mismatched, CN: svn.postgresqlfr.org
-
-
-	Problematic domains:
-
-		- postgresql.eu
-		- svn.postgresqlfr.org	(self-signed)
-		- postresql.org		(404)
-
 -->
 <ruleset name="PostgreSQL">
 
-	<target host="postgresql.eu" />
-	<target host="www.postgresql.eu" />
 	<target host="postgresql.org" />
-	<target host="*.postgresql.org" />
-	<target host="postgresql.us" />
-	<target host="*.postgresql.us" />
+	<target host="www.postgresql.org" />
+	<target host="archives.postgresql.org" />
+	<target host="babel.postgresql.org" />
+	<target host="buildfarm.postgresql.org" />
+	<target host="commitfest.postgresql.org" />
+	<target host="doxygen.postgresql.org" />
+	<target host="ftp.postgresql.org" />
+	<target host="git.postgresql.org" />
+	<target host="jdbc.postgresql.org" />
+	<target host="lists.postgresql.org" />
+	<target host="media.postgresql.org" />
```

Comment:
Can you add a test url please? http://media.postgresql.org/sfpug/instagram_sfpug.pdf